bigquery/deploy/README.md

```diff
@@ -40,7 +12,13 @@
     gcloud container clusters get-credentials elasticsearch-cluster --zone MY_ZONE
     ```
     This will make `kubectl` use this cluster.
-  * Run [kubectl commands](https://github.com/pires/kubernetes-elasticsearch-cluster#deploy)
+
+* Run Elasticsearch on GKE
+  * Deploy Elasticsearch:
+    ```
+    cd bigquery/deploy/k8s_es_deploy/
+    sh deploy.sh
```

Comment:
nit change to ./deploy.sh and make file executable

Reply:
Done